airflow/models/dagpickle.py

```diff
@@ -42,7 +42,7 @@ class DagPickle(Base):
     """
 
     id = Column(Integer, primary_key=True)
-    pickle = Column(PickleType(pickler=dill))
+    pickle = Column(PickleType(pickler=cloudpickle))
```

Comment:
I think the changes in the models needs a migration script to keep the DB rows parsable, I'm trying to do something similar [here](https://github.com/apache/airflow/pull/38358).